Greatly improves player security

Rukofamicom opened the pull request:
## About The Pull Request

* Sensitive player information is no longer visible to administrators in-game
* Sensitive player information is no longer stored in server-side logs accessible by administrators
* Disables kicking players on telemetry timeout due to the impact this kick is having on legitimate player experiences
* Information necessary for processing bans is still stored in server-side database, but it is inaccessible without direct database access

Rukofamicom committed:
removes several references to IP and CID

Rukofamicom committed:
removes references from player panel

Includes opening ban panel from player panel

Rukofamicom committed:
removes ability to search by IP/CID

removes auto-filling of unban information with player IP/CID

Rukofamicom committed:
block the viewing of IP/CID via variables window

itsmeow commented:
If someone joins with a matching CID or IP i think it prints it to admin messaging and log.

Rukofamicom commented:
It does not print to logs of any sort anymore, but I think I probably did miss the direct message to admins thanks.

Rukofamicom committed:
Removes access to telemetry variables

Rukofamicom committed:
removes sensitive information from same connection alert

Rukofamicom committed:
Disables kicking on telemetry timeout

Timeout has to be too long to really help the server much anyway, and it is firing false positives.

Rukofamicom commented:
Alright, got those covered now + went ahead and disabled the telemetry timeout kick due to repeated false positives and how harmful they are to player experience given our short round lengths.
